schema/examples/direct/multiple_signers/status-response.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<direct-signature-job-status-response xmlns="http://signering.posten.no/schema/v1">
+    <signature-job-id>1</signature-job-id>
+    <status signer="12345678910">SIGNED</status>
+    <status signer="10987654321">SIGNED</status>
+    <confirmation-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/complete</confirmation-url>
+    <xades-url signer="12345678910">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/1</xades-url>
+    <xades-url signer="10987654321">https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/xades/2</xades-url>
+    <pades-url>https://api.signering.posten.no/api/{sender-identifier}/direct/signature-jobs/1/pades</pades-url>
+</direct-signature-job-status-response>
```

Comment:
For the record: dersom bare én signatar har signert, ser jeg for meg at `status` for de resterende blir `WAITING` e.l. og at `xades-url`-elementet (naturligvis) bare eksisterer for den personen som har signert.